docs/src/literate/hyperelasticity.jl

```diff
@@ -66,9 +78,72 @@ import KrylovMethods, IterativeSolvers
 # and the tangent of ``\mathbf{S}`` as
 #
 # ```math
-# \frac{\partial \mathbf{S}}{\partial \mathbf{C}} = 4 \frac{\partial \Psi}{\partial \mathbf{C}}.
+# \frac{\partial \mathbf{S}}{\partial \mathbf{C}} = 2 \frac{\partial^2 \Psi}{\partial \mathbf{C}^2}.
```

Comment:
4 times was correct. We have 
$$\mathbf{S} := \frac{\partial \Psi}{\partial \mathbf{E}} = 2 \frac{\partial \Psi}{\partial \mathbf{C}}$$ 
and 
$$\mathbb{C} := 2 \frac{\partial \mathbf{S}}{\partial \mathbf{C}} = 4  \frac{\partial^2 \Psi}{\partial \mathbf{C}}$$